etl/steps/data/garden/irena/2023-12-12/renewable_electricity_capacity.meta.yml

```diff
@@ -1,121 +1,59 @@
-# TODO: This file is a duplicate of the previous step. It is not yet used in the dag and should be updated soon.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Energy
+    short_unit: MW
+    unit: megawatts
+    display:
+      numDecimalPlaces: 0
+    description_short: |-
```

Comment:
that's where I'd add what you mean by capacity - eg., `Amount of electricity or energy that can be generated from x `